### ISSUE TITLE
Buttons present in AnimatedVisual Player should have tool-tip.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#170]
**Describe the bug**

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery.
2. Activate AnimatedVisual Player from All Controls Page.
3. Take keyboard focus to 4 buttons present below the image having symbol icon play, pause, skip back etc and observe if the buttons have a tool tip or not.

**Expected behavior**

The 4 buttons should have tool-tip information on keyboard focus.

**Actual behavior**

No tooltip present for these 4 buttons.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/63801623-9676ee80-c8c5-11e9-9fe5-d707d626551e.png)

**Version Info**

Gallery version (found on Settings page): 

Version Info: 1.2.6.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/20820656)
